knowledge-base/grid-hide-header-row.md

```diff
@@ -0,0 +1,77 @@
+---
+title: How to Remove the Grid Header Row
+description: How to hide the header row of the Grid and TreeList with CSS.
+type: how-to
+page_title: How to Hide the Grid Header Row
+slug: grid-hide-header-row
+position: 
+tags: grid, treelist, hide, header, row
+ticketid: 1467583
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor, TreeList for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+How do I hide the Grid header row?
+
+## Solution
+
+To hide the header row in a Grid or TreeList, set the CSS property value of the header element `k-grid-header` `display` to `none`. 
+
+>To avoid affecting all Grid/TreeList components, apply a specific class to the targeted Grid or TreeList and use the class in the selector.
+
+>caption Hide the Grid/TreeList header row
+
+````CSHTML
+@* Hide the header row of the Grid *@
+
+<style>
+    .no-headers-grid > div > div.k-grid-header {
```

Comment:
```suggestion
    .no-headers-grid .k-grid-header {
```

This selector is too specific and depends on an exact hierarchy that may or may not be present in a future version. The selector I am proposing should be specific enough that a collision is unlikely, yet, is likely to be valid for longer.